src/style.scss

```diff
@@ -0,0 +1,126 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:regular,500,700&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif ;
+  margin: 0;
+  --сolor-white: rgb(255, 255, 255);
+  --color-blue: rgba(0, 172, 220, 1);
+  --color-gray: rgba(97, 96, 112, 1);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 5px;
+  padding: 0 16px;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
```

Comment:
Use HEX instead of RGBA. This is a more concise notation, and it doesn't bloat the code